test.py

```diff
@@ -0,0 +1,38 @@
+import cv2
```

Comment:
Maybe we should leave this file off the repo, and write a more generalized main.py or something. This is more just like one specific test for one sample.